app/helpers/lookup_keys_helper.rb

```diff
@@ -75,7 +75,8 @@ def lookup_key_with_diagnostic(obj, lookup_key, lookup_value)
       :name => "#{lookup_value_name_prefix(lookup_key.id)}[value]",
       :disabled => !lookup_key.overridden?(obj) || lookup_value.use_puppet_default,
       :inherited_value => inherited_value,
-      :lookup_key => lookup_key)
+      :lookup_key => lookup_key,
+      :lookup_key_type => lookup_key.key_type)
```

Comment:
No need to pass yet another parameter, just call `.key_type` on options[:lookup_key] on the receiver